app/Http/Controllers/Users/UsersController.php

```diff
@@ -671,4 +779,17 @@ public function sendPasswordReset($id)
 
         return redirect()->back()->with('error', 'User is not activated, is LDAP synced, or does not have an email address ');
     }
+
+    /**
+     * get user org list with input user ID is the manager
+     *
+     * @author A. Rahardianto
+     * @since [v6.1.0]
+     * @return \Illuminate\Http\RedirectResponse
+     */
+    public function getUserOrg($id)
+    {
+
+        return response()->json(Helper::formatStandardApiResponse('error', null, $assetMaintenance->getErrors()));
```

Comment:
I'm not sure what `$assetMaintenance->getErrors())` would be doing here?

Reply:
oops, it's a garbage code. I forgot to remove these lines.
I will remove it